examples/deserialize-transaction/src/example.ts

```diff
@@ -0,0 +1,408 @@
+/**
+ * EXAMPLE
+ * Deserialize and inspect a transaction with @solana/web3.js.
+ *
+ * Before running any of the examples in this monorepo, make sure to set up a test validator by
+ * running `pnpm test:live-with-test-validator:setup` in the root directory.
+ *
+ * To run this example, execute `pnpm start` in this directory.
+ */
+import createLogger from '@solana/example-utils/createLogger.js';
+import {
+    Address,
+    appendTransactionMessageInstructions,
+    assertAccountDecoded,
+    assertAccountExists,
+    assertIsInstructionWithAccounts,
+    assertIsInstructionWithData,
+    compressTransactionMessageUsingAddressLookupTables,
+    createKeyPairSignerFromBytes,
+    createSolanaRpc,
+    createTransactionMessage,
+    decodeTransactionMessage,
+    fetchJsonParsedAccount,
+    getAddressEncoder,
+    getBase64EncodedWireTransaction,
+    getBase64Encoder,
+    getCompiledTransactionMessageDecoder,
+    getTransactionDecoder,
+    partiallySignTransactionMessageWithSigners,
+    pipe,
+    setTransactionMessageFeePayer,
+    setTransactionMessageLifetimeUsingBlockhash,
+    verifySignature,
+} from '@solana/web3.js';
+import { getAddMemoInstruction, MEMO_PROGRAM_ADDRESS, parseAddMemoInstruction } from '@solana-program/memo';
+import {
+    getTransferSolInstruction,
+    identifySystemInstruction,
+    parseTransferSolInstruction,
+    SystemInstruction,
+} from '@solana-program/system';
+
+const log = createLogger('Deserialize');
+
+/**
+ * SETUP: LOOKUP TABLE ADDRESS
+ * Our fixtures include a lookup table, which we're going to use for this transaction
+ * This is the address of the lookup table account from `scripts/fixtures/address-lookup-table-account.json`
+ */
+const LOOKUP_TABLE_ADDRESS = '2JPQuT3dHtPjrdcbUQyrrT4XYRYaWpWfmAJ54SUapg6n' as Address;
```

Comment:
We should create a new one of these instead of reusing an account that's also used by tests. I should be able to delete the test and the fixture, or delete the example and the fixture without breaking either.

Consider cloning this fixture and naming it `example-deserialize-transaction-address-lookup-table.json`?